haa_downloader.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#HAA Binary downloader By AJAX
+download_haa(){
+        echo "--------------------------------------------------------"
+        echo ""
+        validation=$(curl -s "https://api.github.com/repos/RavenSystem/haa/releases/tags/$version" | grep -Po '(?<="message": ")[^"]*')
+        if [ -z "$validation" ]
+        then
+                if [ ! -d "$version" ]; then
+                        mkdir "$version"
+                fi
+                for i in "${files[@]}"
+                do
+                        file_name="$(basename $i)"
+                        echo -n "Saving ./$version/$file_name ... "
+                        wget -qO "$version/$file_name" "$i"
```

Comment:
Better use curl instead wget

Reply:
Started with just wget calls to the latest links before I decided to add in other stuff, I’ll update it to use curl commands and include something in the header

Reply:
replaced wget with curl,
reduced github api calls to 2 and included a count
included a gpl license statement in the code and printed to terminal